packages/cosmic-swingset/Makefile

```diff
@@ -63,7 +60,7 @@ scenario2-setup:
 	echo 'mmmmmmmm' | $(AGC) gentx --home-client=t1/$(BASE_PORT)/ag-cosmos-helper-statedir --name=ag-solo --amount=1000000stake
 	$(AGC) collect-gentxs
 	$(AGC) validate-genesis
-	./setup/set-json.js ~/.ag-chain-cosmos/config/genesis.json --agoric-genesis-overrides
+	../deployment/set-json.js ~/.ag-chain-cosmos/config/genesis.json --agoric-genesis-overrides
```

Comment:
This is the only bit that worries me, having this package be dependent upon something outside the package subdirectory. But I can't think of a clean solution for it right now, so I guess let's land it this way and then worry about it later.

Reply:
As we do the refactorings in #263 and #317, we can address this better.